contracts/libraries/RewardLib.sol

```diff
@@ -18,28 +18,14 @@ library RewardLib {
     int32 ratingToReward;
   }
 
-  struct StatusRewardContainer {
-    mapping(address => mapping(uint16 => StatusReward)) statusReward;
-  }
-
-  struct StatusReward {
-    bool isPaid;
-  }
-
   struct UserRewards {
```

Comment:
I think it should be renamed to make it clear that this are rewards for just a single period.

`UserRewards` -> `UserPeriodRewards`